api/URL.json

```diff
@@ -0,0 +1,538 @@
+{
+  "api": {
+    "URL": {
+      "__compat": {
+        "mdn_url": "https://developer.mozilla.org/docs/Web/API/URL",
+        "support": {
+          "chrome": [
+            {
+              "version_added": "2",
+              "alternative_name": "webkitURL",
+              "notes": "This feature is implemented under the non-standard name webkitURL."
+            },
+            {
+              "version_added": "32"
```

Comment:
Can you swap this around to be reverse chronological, please? 
[
  { version_added: 32  },
  { version_added: 2, alternative_name: webkitURL}
]